cudaaligner/src/cudaaligner.cpp

```diff
@@ -29,6 +30,7 @@ namespace cudaaligner
 StatusType Init()
 {
     initialize_logger(claraparabricks::genomeworks::logging::LogLevel::warn);
+    GW_CU_CHECK_ERR(cudaFree(nullptr)); // initialize CUDA runtime
```

Comment:
Makes sense, I'm just wondering why is it necessary

Reply:
Not strictly necessary, but since we have an `Init()` function, we may also actually do the initialization.